tools/test/h5dump/CMakeTests.cmake

```diff
@@ -412,7 +412,7 @@
   # --------------------------------------------------------------------
   HDFTEST_COPY_FILE("${HDF5_TOOLS_DIR}/testfiles/tbin1.ddl" "${PROJECT_BINARY_DIR}/testfiles/std/tbin1LE.ddl" "h5dump_std_files")
 
-  if (WIN32)
+  if (WIN32 AND CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION VERSION_LESS 10.0.18362.0)
```

Comment:
Can this be 10.0 or does it have to be the full version number? I thought CMake was usually pretty graceful about not having full revision numbers.

Reply:
Has to be the full version.